contrib/container-images/centos-8-pypy.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -x
```

Comment:
nit: just noticed the filename, can you fix name to pypi ?
pypi: python package index
pypy: a fast python interpreter